TodoList/bot/api/graphClient.ts

```diff
@@ -0,0 +1,38 @@
+// Import polyfills for fetch required by msgraph-sdk-javascript.
+import "isomorphic-fetch";
+import { TeamsFx, createMicrosoftGraphClient, IdentityType } from "@microsoft/teamsfx";
+import { Client, ResponseType }from "@microsoft/microsoft-graph-client"
+
+interface GraphResponse {
+    status: number;
+    body: { [key: string]: any };
+}
+
+export default async function graphRun(AADId: string, token: string) {
```

Comment:
change funtion name to getUserPhoto?
Also `AADId` parameter can be changed to `aadObjectId`

Reply:
The response type can also be updated